apps/src/StudioApp.js

```diff
@@ -308,6 +309,7 @@ StudioApp.prototype.init = function (config) {
       sendToPhone: config.sendToPhone,
       twitter: config.twitter,
       app: config.app,
+      noHowItWorks: config.noHowItWorks,
```

Comment:
I wonder if we should eventually drive this from a user permissions rule; if we've got gamelab locked down but want shares to be public, maybe we auto-hide the How It Works button for users that don't have permission to access gamelab view/edit pages?